src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -200,8 +204,51 @@ extension ExposureDetectionViewController {
 			checkButton.isEnabled = true
 		case .manual:
 			footerView.isHidden = false
-			checkButton.setTitle(AppStrings.ExposureDetection.buttonRefresh, for: .normal)
-			checkButton.isEnabled = riskProvider.manualExposureDetectionState == .possible
+
+			let nextRefresh = riskProvider.nextExposureDetectionDate()
+			let now = Date()
+
+			if nextRefresh > now, countdown == nil {
+				scheduleCountdownTimer(to: nextRefresh)
+			} else if let countdown = countdown, countdown.end != nextRefresh {
+				// Make sure to schedule new countdown if the next refresh time has changed.
+				scheduleCountdownTimer(to: nextRefresh)
+			} else if nextRefresh - 1 > now {
```

Comment:
why this arbitrary -1? whats the reason?

Reply:
![What was the reason?](https://media.giphy.com/media/fGOjgWRzQkC2sHHnq7/giphy.gif)

Reply:
this prevents the countdown to show 00:00:00 as an explicit value and directly enables the button. looks nicer imo